.github/workflows/test.yml

```diff
@@ -0,0 +1,24 @@
+name: Integration Test
```

Comment:
Let's change the name so it is clear what the action is doing. i.e. Runs Rsync Example as Integration Test.